cf_units/tests/test_coding_standards.py

```diff
@@ -75,7 +75,7 @@ def last_change_by_fname():
         """
```

Comment:
(not for this MR) seems to me that this entire test file should be replaced with a pre-commit hook. (perhaps make an issue and move on?)

Reply:
Interesting idea!  I think that could solve Cartopy’s problem at https://github.com/SciTools/cartopy/pull/2278.

Reply:
I opened the issue in Iris as that tends to lead the SciTools repos (except Cartopy).